server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -3691,6 +3691,7 @@ public UsageRecordResponse createUsageResponse(Usage usageRecord, Map<String, Se
                     builder.append(" for VM ").append(vmInstance.getHostName()).append(" (").append(vmInstance.getUuid()).append(") ")
                             .append("with size ").append(usageRecord.getVirtualSize());
                 }
+                usageRecResponse.setSize(usageRecord.getVirtualSize());
```

Comment:
Should this be outside `if (!oldFormat)` block?

Reply:
Nice catch. done.

Reply:
Done